src/bitmessagecli.py

```diff
@@ -617,59 +645,61 @@ def saveFile(fileName, fileData): #Allows attachments and messages/broadcats to
 
     if not os.path.exists(directory):
         os.makedirs(directory)
-        
-    filePath = directory +'/'+ fileName
+
+    filePath = directory + '/' + fileName
```

Comment:
Maybe `os.path.join(directory, fileName)` would be better

Reply:
Added to further manual work ticket